zerocloud/configparser.py

```diff
@@ -872,6 +872,7 @@ def __init__(self, id=None, name=None, exe=None, args=None, env=None,
         self.access = ''
         self.exe_name = exe_name
         self.data_in = False
+        self.location = location or None
```

Comment:
Got it. Can you document those details? 1) location must be a string; 2) it should not be empty, or else the location is disregarded.